addons/l10n_cl/views/res_partner.xml

```diff
@@ -22,4 +19,14 @@
         </field>
     </record>
 
+    <record id="address_street_extended_city_form_l10n_cl" model="ir.ui.view">
+        <field name="name">partner.form.address.extended.city_id.l10n_cl</field>
+        <field name="inherit_id" ref="base_address_extended.address_street_extended_city_form"/>
+        <field name="model">res.partner</field>
+        <field name="arch" type="xml">
+            <field name="city_id" position="attributes">
+                <attribute name="placeholder">Commune</attribute>
```

Comment:
This should be in a primary view explicitly for Chile with an address_view_id specific for Chile otherwise other countries will be affected.  It was incorrect before as view_move_form above should have been done through the address_view_id.  Better to fix it.

Reply:
@prep-odoo